src/Home.tsx

```diff
@@ -16,8 +16,12 @@ class Home extends React.Component {
 		return (
 			<Wrap>
 				<Switch>
-					<Route path="/" exact component={Setlists}/>
-					<Route path="/setlists" exact component={Setlists} />
+					<Route path="/" exact component={LatestShows}/>
+					<Route path="/shows" exact component={Shows} />
+					<Route path="/shows/:id" exact component={Shows} />
+					<Route path="/shows/year/:year" exact component={Shows} />
+					<Route path="/shows/venue/:venue_id" exact component={Shows} />
+					<Route path="/shows/state/:state" exact component={Shows} />
```

Comment:
You could put these routes in the `Shows` component to make the Home route less bloated.

Reply:
i'll circle back on that -- not sure how to get render working on that page.